lang/cs/Org.Apache.REEF.Common/Telemetry/MetricsService.cs

```diff
@@ -116,14 +116,18 @@ public void OnError(Exception error)
 
         /// <summary>
         /// Observer of IDriverMetrics.
-        /// When Driver metrics data is changed, this method will be called
+        /// When Driver metrics data is changed, this method will be called.
+        /// It calls Sink to store/log the metrics data.
         /// </summary>
-        /// <param name="value"></param>
-        public void OnNext(IDriverMetrics value)
+        /// <param name="driverMetrics">driver metrics data.</param>
+        public void OnNext(IDriverMetrics driverMetrics)
         {
-            var list = new List<KeyValuePair<string, string>>();
-            list.Add(new KeyValuePair<string, string>("SystemState", value.SystemState));
-            list.Add(new KeyValuePair<string, string>("TimeUpdated", value.TimeUpdated.ToLongTimeString()));
+            var list = new List<KeyValuePair<string, string>>()
+            {
+                { new KeyValuePair<string, string>("SystemState", driverMetrics.SystemState) },
```

Comment:
No need to write `new KeyValuePair...` explicitly - it can be simply
`{ "SystemState", driverMetrics.SystemState }`
(see my earlier example)

Reply:
I tried, it gives build error.

Reply:
Oops, I meant this:
```C#
Sink(new Dictionary<string, string>()
{
    { "SystemState", driverMetrics.SystemState },
    { "TimeUpdated", driverMetrics.TimeUpdated.ToLongTimeString() }
});
```